src/main/java/org/folio/qm/util/StatusUtils.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.qm.util;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+import lombok.experimental.UtilityClass;
+
+import org.folio.qm.domain.entity.RecordCreationStatus;
+import org.folio.qm.domain.entity.RecordCreationStatusEnum;
+import org.folio.qm.domain.entity.RecordCreationStatusUpdate;
+
+@UtilityClass
+public class StatusUtils {
+
+  public static RecordCreationStatus getStatusNew(UUID qmParsedRecordId, String jobExecutionId) {
+    final RecordCreationStatus status = new RecordCreationStatus();
+    status.setId(qmParsedRecordId);
+    status.setJobExecutionId(UUID.fromString(jobExecutionId));
+    status.setStatus(RecordCreationStatusEnum.NEW);
+    status.setCreatedAt(new Timestamp(System.currentTimeMillis()));
+    status.setUpdatedAt(new Timestamp(System.currentTimeMillis()));
```

Comment:
There is no need to populate status with values for createdAt, updatedAt, and status when saving new status. These values will be populated on DB-side